data.bloodpac.org/portal/gitops.json

```diff
@@ -166,8 +198,11 @@
   },
   "requiredCerts": ["security_quiz"],
   "featureFlags": {
-    "explorer": true
+    "explorer": true,
+    "analysis": true,
+    "explorerPublic": true
```

Comment:
by setting this and set guppy in private mode, if user is not logged in and they go to explorer, they can only see an empty explorer